ecosystem/typescript/sdk/src/token_client.ts

```diff
@@ -165,8 +261,20 @@ export class TokenClient {
     return tableItem;
   }
 
-  // Retrieve the token's creation_num, which is useful for non-creator operations
-  async getTokenBalance(creator: MaybeHexString, collectionName: string, tokenName: string): Promise<number> {
+  /**
+   * Queries specific token from account's TokenStore
+   * @param creator Hex-encoded 16 bytes Aptos account address which created a token
+   * @param collectionName Name of collection, which holds a token
+   * @param tokenName Token name
+   * @returns Token in next form
```

Comment:
is this correct? `in next form`

Reply:
I tried to pay attention to a Token object interface, that is below.

Reply:
`@returns Token object in below format` ?